### PR TITLE
Added the email input field

### LINKED ISSUE #4: Create Input
image

- Should pull from shadcn

### PR DESCRIPTION
**Pull request by darsh175223**
- Has a " An interface declaring no members is equivalent to its supertype" error at src\components\ui\input.tsx. Added email input field and corresponding label

### CODE REVIEW

File: src/components/applications/application.tsx [hunk 1]
```diff
@@ -26,16 +28,10 @@ const Application = ({ title, questions }: props) => {
...
-        meta={{
-          type: "text",
-          title: "Email",
-          placeholder: "Enter an Email",
-          maxLength: 250,
-          value: "SAMPLE EMAIL",
-          disabled: true,
-        }}
-      />
+      <div>
```
Comment (seanquiambao) [inline src/components/applications/application.tsx]: add small gap between label and input

File: src/components/applications/application.tsx [hunk 2]
```diff
@@ -26,16 +28,10 @@ const Application = ({ title, questions }: props) => {
...
+        <Label htmlFor="email">Email</Label>
```
Comment (seanquiambao) [inline src/components/applications/application.tsx]: should be in its own component.

you can move this code to /components/global/inputs/input.tsx

**Review by seanquiambao — CHANGES_REQUESTED** [review]
link your issue to this pr and add a screenshot of the component.

File: src/components/applications/application.tsx (hunk 2 above)
Comment (darsh175223) [reply src/components/applications/application.tsx]: Wait so I'm a bit confused on what I should do. Should I make a seperate component that contains both a label and input field?

**Comment by darsh175223** [comment]
Also here's a screenshot of the label and input field.
image

File: src/components/applications/application.tsx (hunk 2 above)
Comment (seanquiambao) [reply src/components/applications/application.tsx]: yeah there should be a separate component containing both label and input.

but you don't need to create a new file for it, there's already a file to put your code in.

Comment (seanquiambao) [reply src/components/applications/application.tsx]: https://github.com/acm-ucr/starlight/blob/dev/src/components/global/inputs/input.tsx 
in here

**Review by shahdivyank — APPROVED** [review]

**Pull request merged**